CommandLine/CommandLine.swift

```diff
@@ -569,7 +577,7 @@ public class CommandLine {
       let format = formatOutput != nil ? formatOutput! : defaultFormat
 
       let name = _arguments[0]
-      print(format("Usage: \(name) [options]", .About), terminator: "", toStream: &to)
+      print(format("Usage: \(name) [options]" + (self.suplementalHelp ?? ""), .About), terminator: "", toStream: &to)
```

Comment:
On second thought, I'll leave this in.  

I'll put in an additional var.  One var for the appending the end of the usage message, the other for prepending to the start of the help section below the usage.